stetho/src/main/java/com/facebook/stetho/common/ListAccumulator.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.common;
+
+import java.util.List;
+
+public interface ListAccumulator<E> extends List<E>, Accumulator<E> {
```

Comment:
Standard Java convention perhaps, but not necessarily Android convention.  One of the main motivators for this is the glaring performance implications of LinkedList's (and let's be honest, there are only two choices here *grin*).  LinkedList's, as they're implemented in the VM, are so incredibly memory inefficient that you should rule out their usage on mobile in all conceivable workloads.  In this way, you can assume all List instances in use are in fact ArrayList.  If the assumption fails, you should convert the code :)

My argument is fundamentally based on this fact: `ListAccumulator` can have only one reasonable implementation.